Update sk.json

matoweb opened the pull request:
Hi, I´ve just finished translation to Slovak language. Hopefully, it will help a bit.

Matus

matoweb committed:
Update sk.json

Hi, I´ve just finished translation to Slovak language. Hopefully, it will help a bit.

Matus

AnnaTomanek commented on lang/sk.json:
```diff
@@ -35,103 +35,103 @@
 			"12": "Nepodporovaný typ súboru.",
-			"13": "[MISSING]The connector configuration file is invalid.",
-			"14": "[MISSING]Invalid connector plugin: {pluginName}.",
+			"13": "Konfigurácia súboru je neplatná.",
```
There seems to be a small misunderstanding here. This should really mean: "The configuration file of the CKFinder server-side connector is invalid.".

AnnaTomanek commented on lang/sk.json:
```diff
@@ -145,141 +145,143 @@
 	"chooseResizedImage": {
-		"title": "[MISSING]Choose Resized",
-		"originalSize": "[MISSING]Original Size",
+		"title": "Zmeniť veľkosť",
```
This is actually a caption for the button that lets you select (choose) an image, resize it, and then pass the resized one to another application (like CKEditor)-> hence "Choose Resized". The full version would probably be "Choose resized version of an image".

AnnaTomanek commented on lang/sk.json:
```diff
@@ -145,141 +145,143 @@
+		"downloadAction": "Sťahujem originálny súbor.",
+		"keepAspectRatio": "Udržať polomer",
+		"loading": "Nahrávam obrázok",
+		"presets": "Zmeniť",
```
"Presets" is a noun here, these are pre-defined settings/configurations for image editor filters.

AnnaTomanek commented on lang/sk.json:
```diff
@@ -145,141 +145,143 @@
+			"vibrance": "Citlivosť"
 		},
+		
+		//It doesn´t need to be translated.
```
Please don't add additional comments directly into the file - just remove the `[MISSING]` part and it's clear. Thanks!

AnnaTomanek commented:
Hi Matus, thank you so much for your amazing work! I added a few comments to your file, can you please review and if needed, propose some changes? Your help is much appreciated!

matoweb committed:
Update sk.json

Hi
I´ve updated your requested lines. It should be OK now, if something, just let me know :)
Matus

AnnaTomanek commented on lang/sk.json:
```diff
@@ -145,141 +145,143 @@
 	"chooseResizedImage": {
-		"title": "[MISSING]Choose Resized",
-		"originalSize": "[MISSING]Original Size",
+		"title": "Vybrať upravenú verziu obrázku",
```
Any chance to make it shorter? This is a toolbar button label, so it should be as compact as possible :(
![image](https://cloud.githubusercontent.com/assets/538321/14279524/ef7cf3d0-fb2d-11e5-8d42-81e602d5f5cc.png)

AnnaTomanek commented:
Hi Matus, thanks for looking into this. Just one more comment/change request and we should be good to go. Sorry about any inconvenience caused!

matoweb committed:
Update sk.json

Hey, I´ve checked it, and made it bit shorter for you :) everybody in Slovakia will clearly understand it. If something is not clear, just let me know.

If you have any other software or whatever to translate, just let me know and I can do it for you.
Matus

AnnaTomanek commented:
Hi Matus, thanks a lot! Sorry it took me a while to come back to this :)

As promised, we have a free CKFinder 3 license for anyone that provides a significant translation update. If you ever need such a license, just [contact us](http://cksource.com/contact) and send the URL that your CKFinder installation will be used at and we'll provide you the license key.

By the way, you can subscribe to issue #57 in order to be notified when new translations are needed. I'll be updating this thread before releases, whenever there are new language strings to be translated - by subscribing to this issue you will be automatically notified about this.

If you wanted to help translate CKEditor, we are doing it through [Transifex](https://www.transifex.com/ckeditor/ckeditor/). Feel free to join, you can find more information about the CKEditor localization process here: http://docs.cksource.com/CKEditor_3.x/Developers_Guide/Localization

Thanks again for your help!
